### PR TITLE
Syntax: add clang-format

### PR DESCRIPTION
**Pull request by TruncatedDinosour**
I just mainly added it for myself, you
can choose if merge or not, it's just nice
to see the config file highlighted

Not an important feature, just nice I guess lol

Just close this PR if you decide not to merge :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - New style component `header-filesize` to show size of the displayed file in the header. See #1988 (@mdibaiee)
 - Use underline for line highlighting on ANSI, see #1730 (@mdibaiee)
+- Mapped clang-format config file (.clang-format) to TOML syntax (@TruncatedDinosour)
```
Comment (keith-hall) [inline CHANGELOG.md:7]: in the changelog you wrote TOML, in the syntax mapping you wrote YAML :sweat_smile:

**Comment by Enselic** [comment]
`time` fails to build in CI

we probably just need https://github.com/trishume/syntect/pull/422 in bat too

**Comment by sharkdp** [comment]
Thank you very much for your contribution!

Why does this bump the bat version and update dependencies? Could you please revert this?

File: CHANGELOG.md (hunk 1 above)
Comment (TruncatedDinosour) [reply CHANGELOG.md:7]: > in the changelog you wrote TOML, in the syntax mapping you wrote YAML sweat_smile

whoops

**Comment by TruncatedDinosour** [comment]
> Thank you very much for your contribution!
> 
> Why does this bump the bat version and update dependencies? Could you please revert this?

sure

**Comment by TruncatedDinosour** [comment]
Messed something up wait

**Comment by TruncatedDinosour** [comment]
Okay now it should be fine

**Review by TruncatedDinosour — COMMENTED** [review]
- Changelog: fixed
- Cargo files: reset

**Comment by sharkdp** [comment]
Thanks!

**Pull request merged**